### PR TITLE
Add a README

### PR DESCRIPTION
**Pull request by as2388**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+   The exact version required by this project is checked out automatically.
+ - `rapid-pro-server` is the root address of the RapidPro server to retrieve data from e.g. `https://app.rapidpro.io`.
+ - `rapid-pro-token` is the access token for this instance of RapidPro. The access token may be found by logging into 
+   RapidPro's web interface, then navigating to your organisation page (via the button in the top-right), then copying
+   the hexadecimal string given after "Your API Token is ..."
+ - `data-root` is an absolute path to the directory in which all pipeline data should be stored.
+   Raw data will be saved to TracedData JSON files in `<data-root>/Raw Data`. 
+
+### 3. Generate Outputs
+Finally, process the raw data to produce outputs for The Interface, ICR, Coda, and messages/individuals/production
```
Comment (lukechurch) [inline README.md]: I don't think we're producing artefacts for The Interface still?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+As well as uploading the messages, individuals, and production CSVs to Drive, this stage outputs the following to
+`<data-root>/Outputs`:
+ - Local copies of the messages, individuals, and production CSVs (`csap_mes.csv`, `csap_ind.csv`, `csap_prod.csv`)
+ - A serialized export of the list of TracedData objects representing all the data that was exported for analysis 
+   (`traced_data.json`)
+ - For each week of radio shows, a random sample of 200 messages that weren't classified as noise, for use in ICR (`ICR/`)
+ - Coda V2 messages files for each dataset (`Coda Files/<dataset>.json`). To upload these datasets to Coda, use the 
+   `set.py` or `add.py` tools in the [Coda V2](https://github.com/AfricasVoices/CodaV2/tree/master/data_tools) repository.
+ 
+To make coded data available to the pipeline, use the `get.py` script in `CodaV2/data_tools` to export coded messages
```
Comment (lukechurch) [inline README.md:89]: For another day - should we consider integrating these into the pipeline as well?

**Review by lukechurch — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (as2388) [reply README.md]: We're not; removed.

(Once the other open PRs are merged I will need to re-read the readme and make sure it's all still up-to-date)

File: README.md (hunk 2 above)
Comment (as2388) [reply README.md:89]: There was another PR which includes this as a run script. Will update the README to reflect that.

Does that suffice or are you getting at pushing directly from within the pipeline's main python script?

**Pull request merged**